cli/cmd/alert_profiles.go

```diff
@@ -209,6 +230,105 @@ func buildAlertProfileDetailsTable(profile api.AlertProfile) string {
 	return detailsTable.String()
 }
 
+func promptUpdateAlertProfile(args []string) (api.AlertProfileResponse, error) {
+	var (
+		msg       = "unable to update alert profile"
+		profileID string
+		err       error
+	)
+	if len(args) == 0 {
+		profileID, err = promptSelectProfile()
+		if err != nil {
+			return api.AlertProfileResponse{}, errors.Wrap(err, msg)
+		}
+	} else {
+		profileID = args[0]
+	}
+
+	var existingProfile api.AlertProfileResponse
+	cli.StartProgress("Retrieving alert profile...")
+	err = cli.LwApi.V2.Alert.Profiles.Get(profileID, &existingProfile)
+	cli.StopProgress()
+	if err != nil {
+		return api.AlertProfileResponse{}, errors.Wrap(err, msg)
+	}
+
+	if err != nil {
+		return api.AlertProfileResponse{}, err
+	}
+
+	queryYaml, err := yaml.Marshal(existingProfile.Data.Alerts)
+	if err != nil {
+		return api.AlertProfileResponse{}, errors.Wrap(err, msg)
+	}
+
+	prompt := &survey.Editor{
+		Message:       fmt.Sprintf("Update alert templates for profile %s", profileID),
+		Default:       string(queryYaml),
+		HideDefault:   true,
+		AppendDefault: true,
+		FileName:      "templates*.yaml",
+	}
+	var templatesString string
+	err = survey.AskOne(prompt, &templatesString)
+	if err != nil {
+		return api.AlertProfileResponse{}, errors.Wrap(err, msg)
+	}
+
+	var templates []api.AlertTemplate
+	err = yaml.Unmarshal([]byte(templatesString), &templates)
+	if err != nil {
+		return api.AlertProfileResponse{}, errors.Wrap(err, msg)
+	}
+
+	cli.StartProgress(" Updating alert profile...")
+	response, err := cli.LwApi.V2.Alert.Profiles.Update(profileID, templates)
+
+	cli.StopProgress()
+	return response, err
+}
+
+func promptSelectProfile() (string, error) {
+	profileResponse, err := cli.LwApi.V2.Alert.Profiles.List()
+	if err != nil {
+		return "", err
+	}
+	var profileList = make([]string, 0)
+	for _, p := range profileResponse.Data {
+		if len(p.Alerts) >= 1 {
+			profileList = append(profileList, p.Guid)
+		}
+	}
+
+	sort.Slice(profileList, func(i, j int) bool {
+		return profileList[i] < profileList[j]
+	})
```

Comment:
should we also filter the alert profiles that we can't update, if there are any (you should check)